src/java_bytecode/ci_lazy_methods.h

```diff
@@ -48,6 +48,7 @@ class ci_lazy_methodst:public messaget
     const std::vector<irep_idt> &main_jar_classes,
     const std::vector<irep_idt> &lazy_methods_extra_entry_points,
     java_class_loadert &java_class_loader,
+    const std::vector<irep_idt> &java_load_classes,
     const select_pointer_typet &pointer_type_selector,
     message_handlert &message_handler);
```

Comment:
This constructor already has too many arguments, but I won't request to refactor that since it can be time consuming.